api/src/@types/prisma-json-types/index.ts

```diff
@@ -0,0 +1,11 @@
+// This file must be a module, so we include an empty export.
```

Comment:
I don't know if I would put this file in the @types folder. We only have it in order to override/augment types in the actual @types namespace, which this wouldn't fall under. In the variants branch i had it in a `prismajson.d.ts` file at the top level, but I don't think it ultimately matters where we settle on putting the file or what it's called